### PR TITLE
feat: Add cake price in usd to web page title

### PR DESCRIPTION
**Pull request by memoyil**

### CODE REVIEW

File: src/hooks/useGetDocumentTitlePrice.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { useEffect } from 'react'
+import useGetPriceData from './useGetPriceData'
+
+const useGetDocumentTitlePrice = () => {
+  const priceData = useGetPriceData()
+  const cakePriceUsdNumber = priceData ? Number(priceData.prices.Cake) : Number.NaN
```
Comment (RabbitDoge) [inline src/hooks/useGetDocumentTitlePrice.ts:6]: same feedback than on the FE. Don't use Number

**Review by RabbitDoge — CHANGES_REQUESTED** [review]

File: src/hooks/useGetDocumentTitlePrice.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  const cakePriceUsdString = Number.isNaN(cakePriceUsdNumber)
+    ? ''
+    : ` - $${cakePriceUsdNumber.toLocaleString(undefined, {
+        minimumFractionDigits: 3,
+        maximumFractionDigits: 3,
+      })}`
+
+  useEffect(() => {
+    document.title = `PancakeSwap${cakePriceUsdString}`
+  })
```
Comment (RabbitDoge) [inline src/hooks/useGetDocumentTitlePrice.ts:17]: this effect will rerun on each render, causing some serious performances issues

File: src/hooks/useGetDocumentTitlePrice.ts (hunk 1 above)
Comment (memoyil) [reply src/hooks/useGetDocumentTitlePrice.ts]: done, i added bignumber library

File: src/hooks/useGetDocumentTitlePrice.ts (hunk 2 above)
Comment (memoyil) [reply src/hooks/useGetDocumentTitlePrice.ts]: done

File: package.json [hunk 3]
```diff
@@ -34,6 +34,7 @@
     "@web3-react/walletconnect-connector": "^6.1.1",
     "@web3-react/walletlink-connector": "^6.0.9",
     "ajv": "^6.12.3",
+    "bignumber.js": "^9.0.0",
```
Comment (RabbitDoge) [inline package.json:37]: Adding `bignumber.js` for this , really ? 😅 
The exchange use JSBI from the pancake-sdk.

**Review by RabbitDoge — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (memoyil) [reply package.json:37]: I couldn't find how to use it for price it only contains integer functions as far as i see, can you give me an example?

**Review by RabbitDoge — APPROVED** [review]

**Pull request merged**